open_event/static/js/jquery/jquery.ellipsis.js

```diff
@@ -0,0 +1,45 @@
+// Borrowed from http://stackoverflow.com/a/1022672/1562480
+(function ellipsisInit($) {
+    $.fn.ellipsis = function () {
+        return this.each(function () {
+            var el = $(this);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected `this`.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2657214412/issues/source?bid=3233864&fileBranchId=3324174#l5)